web/src/narrow.js

```diff
@@ -423,7 +423,13 @@ export function activate(raw_operators, opts) {
     // Open tooltips are only interesting for current narrow,
     // so hide them when activating a new one.
     $(".tooltip").hide();
-
+    if (
+        narrow_state.filter() !== undefined &&
+        filter !== undefined &&
+        JSON.stringify(narrow_state.filter()._operators) === JSON.stringify(filter._operators)
```

Comment:
I think this may not be correct -- does this just check the operators, and not the operands? And also, some narrows may define a scroll position, and this doesn't look at that.